oshi-core/src/main/java/oshi/driver/linux/Sysfs.java

```diff
@@ -185,8 +184,7 @@ public static String queryBiosDescription() {
     /**
      * Query the bios version from sysfs
      *
-     * @param biosRevision
-     *            A revision string to append
+     * @param biosRevision A revision string to append
```

Comment:
Here's another point where our formatters differ.   OSHI's format file is in `oshi-dist/src/main/resources/eclipse` if you're able to use it.   It's essentially the "Sun Java Conventions" with spaces instead of tabs and longer line wrap.